blog/2024-12-20-2024-survey-results.md

```diff
@@ -0,0 +1,144 @@
+## Who are you? 
+
+### OpenRefine Communities 
+Overall, our largest communities remain stable, with Researcher (38.50%) becoming the largest following Librarian (34.01%) and Data Science(25.66%), Wikimedian (25.66%), and GLAM (22.57%) close behind.
+
+This year, we introduced OpenSource Intelligence (OSINT) and Civic Tech as default options. We had information that OpenRefine was used by those communities, and we wanted to confirm it. We have 10 CiviTech users and 6 OSINT respondents. So, while they are present and align with our vision of building a more informed world where working with data is easy and engaging, they do not represent the majority of our users.
+
+[![](/img/2024survey/01communities.png)](/img/2024survey/01communities.png)
+
+**Change in methodology**:  In this question, an individual may identify with multiple communities. In the previous year, we normalized the results based on the total number of communities selected. In 2024, 226 respondents identified, on average, with 2.48 communities (we received 561 selections). With our previous approach, we would have represented CiviTech as 1.78% of our user base (10 divided by 561). This year, we computed the results based on the total number of respondents (226), which allows us to say that 4.42% of our users identify as CiviTech users.
+
+### Professional Usage
+
+Last year, 85.1% of respondents indicated that they primarily use OpenRefine for work. This year, we provided a scale to measure the proportion of their usage in a professional environment. The results are consistent with those from 2022, as 71% of respondents reported using OpenRefine primarily in a professional setting (at least 70% of the time).
+
+[![](/img/2024survey/02pro-usage.png)](/img/2024survey/02pro-usage.png)
+
+### Languages used 
+
+The question "In which languages do you use OpenRefine (both its interface and the datasets you work with)?" reveals the diversity of our community, with 37 languages reported. English is less dominant than it was two years ago, dropping from 64.1% to 55.7% this year. German, French, and Spanish remain the top three languages, followed by a long tail of 33 other languages. 
+
+Interestingly, 40.27% of users reported interacting in multiple languages. In our next survey, it may be beneficial to separate questions about the user interface from the dataset language.
+
+[![](/img/2024survey/03languages.png)](/img/2024survey/03languages.png)
+
+### Ovall Satisfaction and Net Promoter Score
+
+In 2022, we asked our users to provide a general score for the software for the first time. On average, respondents rated OpenRefine a solid 8 out of 10!
+
+This year, instead of requesting a numerical score, we calculated a Net Promoter Score (NPS), which ranges from -100 to +100. OpenRefine achieved a total score of 63.72. Among respondents, 72.57% were promoters (those who scored nine or ten) and would recommend OpenRefine, while only 8.85% were detractors (those who scored six or below).
```

Comment:
```suggestion
This year, instead of requesting a numerical score, we calculated a [Net Promoter Score (NPS)](https://en.wikipedia.org/wiki/Net_promoter_score), which ranges from -100 to +100. OpenRefine achieved a total score of 63.72. Among respondents, 72.57% were promoters (those who scored nine or ten) and would recommend OpenRefine, while only 8.85% were detractors (those who scored six or below).
```